datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -591,6 +591,8 @@ def _submit_gauges_from_histogram(self, metric_name, metric, scraper_config, hos
                 )
             elif sample[self.SAMPLE_NAME].endswith("_count"):
                 tags = self._metric_tags(metric_name, val, sample, scraper_config, hostname)
+                if scraper_config['send_histograms_buckets']:
+                    tags = tags + ["upper_bound:none"]
```

Comment:
Just a nitpick but this could be simplified to ```tags += ["upper_bound:none"]```. Other than that, looks good! 👍

Reply:
Let's do this 🙂 

```suggestion
                    tags.append("upper_bound:none")
```